src/Components/sharedComponents/ImageBuilderHeader.tsx

```diff
@@ -122,7 +122,7 @@ export const ImageBuilderHeader = ({
               }
             />
           </FlexItem>
-          {experimentalFlag && (
+          {!inWizard && (
```

Comment:
the buttons were not hidden when inWizardV2 before, only disabled.
This is the only code change